util/decorators.py

```diff
@@ -2,32 +2,26 @@
 from functools import wraps
 
 from flask import abort
-from flask import redirect
 from flask import request
 from flask.helpers import make_response
-from google.appengine.api import users
+from main import oidc
 
+from models.employee import Employee
 from models.access_key import AccessKey
 from util.csrf import check_csrf_protection
 
 
-def user_required(func):
+def admin_required(func):
     @wraps(func)
     def decorated_view(*args, **kwargs):
-        if not users.get_current_user():
-            return redirect(users.create_login_url(request.url))
+        username = oidc.user_getfield('email').split('@')[0]
+        isAdmin = Employee.query(
```

Comment:
How about we reuse `util.auth.is_admin()` here?

Reply:
Completely forgot about that function! Will fix.